cice.setup

```diff
@@ -1176,6 +1224,7 @@ if ( ${dosuite} == 1 ) then
   cat >> ${tsdir}/suite.submit << EOF0
 
 set nonomatch && rm -f ciceexe.* && unset nonomatch
+set nonomatch && rm -f ice_in_save* && unset nonomatch
```

Comment:
why not simply
~~~diff
-set nonomatch && rm -f ciceexe.* && unset nonomatch
+set nonomatch && rm -f ciceexe.* ice_in_save* && unset nonomatch
~~~

Reply:
We can go either way.  This happens once at the end of the test suite, so cost is not an issue.  By splitting the lines up, it's easier to comment out one or the other for testing/debugging of the ciceexe or ice_in_save feature.  Two lines seems a little cleaner, but it doesn't make much difference.  I guess I prefer to leave two lines.